modules/mockk-android/build.gradle.kts

```diff
@@ -11,43 +11,20 @@ description = "Mocking library for Kotlin (Android instrumented test)"
 val mavenName: String by extra("MockK Android")
 val mavenDescription: String by extra("${project.description}")
 
-@Suppress("UnstableApiUsage")
 android {
     packagingOptions {
         resources {
             excludes += "META-INF/LICENSE.md"
             excludes += "META-INF/LICENSE-notice.md"
         }
     }
-
-    defaultConfig {
-        testInstrumentationRunner = "android.support.test.runner.AndroidJUnitRunner"
-        testInstrumentationRunnerArguments["notAnnotation"] = "io.mockk.test.SkipInstrumentedAndroidTest"
```

Comment:
📝This class was not included in the test APK and an error log was displayed, so I remove this `notAnnotation` config.